src/System.Runtime/src/System.Runtime.csproj

```diff
@@ -38,6 +38,10 @@
     <Compile Include="System\Reflection\RuntimeReflectionExtensions.cs" />
     <Compile Include="System\Runtime\CompilerServices\Attributes.cs" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == '' OR '$(TargetGroup)' == 'netstandard1.7' OR '$(TargetGroup)' == 'uap101aot'">
```

Comment:
It looks like it should be covered, but after I remove it such that it just says:
```
Condition="'$(TargetGroup)' == '' OR '$(TargetGroup)' == 'uap101aot'"
```
I'm getting the following build errors:
```
EXEC : DEFAULT_APPNAME error : 0 : Did not find type 'T:System.IO.HandleInheritability' in any of the seed assemblies. [...\src\System.Runtime\src\System.Runtime.csproj]
EXEC : DEFAULT_APPNAME error : 0 : Did not find type 'T:System.Threading.WaitHandleExtensions' in any of the seed assemblies. [...\src\System.Runtime\src\System.Runtime.csproj]
```
I'm not seeing these when building the src project, .builds file, or package projects/builds files manually, only when building from root with `build -debug`. Any idea why?